docs/suspend-cluster.rst

```diff
@@ -0,0 +1,90 @@
+.. _suspend-cluster:
+
+=================
+Suspend a cluster
+=================
+
+This guide will provide a quick overview of how to suspend your cluster using
+the CrateDB Cloud Console. For the purposes of this guide, it is assumed that you have a cluster up and running and can access the Console. If not, please refer to the :ref:`tutorial on how to deploy a cluster for the first time
+<cloud-tutorials:cluster-deployment>`.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _what-is-suspendion:
+
+What is cluster suspension
+==========================
+
+Cluster suspension is a feature that enables you to pause the computation on
+your cluster temporarily. An example situation might be that the project
+you're working on has been put on hold. In that case, you don't need the computation, but you still need the storage to preserve your data. The cost of running a cluster is split into two parts: **Compute** and **Storage**. The benefit here is that while the cluster is suspended, you are only charged for the storage.
```

Comment:
Maybe remove the sentence "in that case ..." as it seems a bit redundant.

Reply:
The rest of the document seems line-wrapped, while these two lines are pretty long - is there some convention here?

Reply:
Ah, my auto-wrapper keeps acting up. Yeah, the convention is line-wrap to 79 chars.